tests/DummyData.spec.js

```diff
@@ -122,6 +122,18 @@ const deprecatedAccount2KeyInfoObjects = () => [{
     },
 }];
 
+const deprecatedAccount2KeyInfos = () => [
+    KeyInfo.fromObject(
+        deprecatedAccount2KeyInfoObjects()[0],
+        true,
+        deprecatedAccount2KeyInfoObjects()[0].legacyAccount.address,
+    ),
+];
+
+const deprecatedAccount2Keys = () => [
+    new Key(secrets[1], false)
+];
```

Comment:
Why are these methods? They could be simply consts, e.g.:
```ts
const deprecatedAccountAsKeyInfo = KeyInfo.fromObject(...)
```
as the tests do not seem to manipulate the data.

Reply:
That is true. They are functions because my initial tries with the tests had dummy data, which was dependent on other function data.

Reply:
Well, actually only the `deprecatedAccount2Keys` can be a `const`, because `deprecatedAccount2KeyInfos` depends on `deprecatedAccount2KeyInfoObjects()`, which depends on `hashes()`, which cannot be calculated before the Nimiq lib is loaded, which is only _after_ the DummyData file has been run.

Reply:
Turns out, even the `deprecatedAccount2Keys` has to be a function, because `new Key()` internally uses `Nimiq` for default address derivation.